### PR TITLE
Team management

### PR DESCRIPTION
**Pull request by mgale456**
I ran `docker-compose exec web python manage.py startapp team_management` again after renaming the directory. Then copied most of what Timur had into the newly created app. I also removed a bunch of unnecessary stuff that only needs to be in the project folder (composeexample) like `settings.py` and `wsgi.py`. Mostly just cleaned up the app folder and project folder to better match the tutorial.

I also cleaned up the static files implementation based on [Django tutorial part 6](https://docs.djangoproject.com/en/2.2/intro/tutorial06/). This is not ready for production, but is standard for development, I think.

The site's css and icons work for me:
image

### CODE REVIEW

**Comment by mgale456** [comment]
Here's a bit more precise changelog:

startapp command added:
 - `apps.py`, `tests.py` and `migrations` that weren't there.
 - I copied Timur's versions of the other files

Moved `templates` into the `team_management` app directory

Fixed `manage.py` to point to `composeexample/settings.py`

I removed:
 - `team_management/settings.py` after updating the changes into `composeexample/settings.py`
 - `team_management/wsgi.py`
 - the command in the `docker-compose.yml` file as mentioned by Logan

fixed url pathings to follow the tutorials in both `composeexample/urls.py` and `team_management/urls.py`
 - This includes changing the handling of static files

Updated `index.html` template for the static files

File: django/team_management/urls.py [hunk 1]
```diff
@@ -15,16 +15,15 @@
 """
 from django.contrib import admin
 from django.urls import path
-from django.conf import settings
-from django.conf.urls.static import static
+#from django.conf import settings
+#from django.conf.urls.static import static
```
Comment (lpulley) [inline django/team_management/urls.py]: No need to comment things out to save for later. If we need to see how something once was, we can go back in git. I'm going to delete and commit these.

**Review by lpulley — CHANGES_REQUESTED** [review]
Need to delete commented lines.

**Review by lpulley — APPROVED** [review]

**Comment by lpulley** [comment]
Apologies for the weirdness there. I had changed modes on files that were deleted, which resulted in a weird type of conflict that GitHub web couldn't handle. I just merged `master` onto `team-management` and then GitHub was able to merge `team-management` onto `master` cleanly.

File: django/team_management/urls.py (hunk 1 above)
Comment (mgale456) [reply django/team_management/urls.py]: Yep. I forgot about those.

**Pull request merged**